nvflare/private/fed/server/server_engine.py

```diff
@@ -240,9 +256,9 @@ def _listen_command(self, listen_port, run_number):
                         replies = self.aux_send(
                             targets=targets, topic=topic, request=request, timeout=timeout, fl_ctx=fl_ctx
                         )
-                        conn.send(replies)
-            except BaseException:
-                self.logger.warning("Failed to process the child process command.")
+                        self._send(conn, replies)
```

Comment:
your send signature is `def _send(self, conn, data, job_id, run_number):`

Reply:
sorry, forgot I change the signature. thanks for catch it